packages/cli/src/cog/cog.stac.job.ts

```diff
@@ -217,11 +217,9 @@ export class CogStacJob implements CogJob {
         bounds: metadata.targetBounds,
         oneCogCovering: ctx.oneCogCovering,
       },
+      processingId: ctx.override?.processingId,
```

Comment:
This fix a problem for passing `processingId` to `job.json`. Otherwise, I need to set it as `job.json.processingId = ctx.override?.processingId;`